x-pack/plugins/observability_solution/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -147,52 +148,48 @@ export const createLogThresholdExecutor = (libs: InfraBackendLibs) =>
           ? actions.reduce((next, action) => Object.assign(next, action.context), {})
           : {};
 
-      const alert = alertWithLifecycle({
-        id,
-        fields: {
-          [ALERT_EVALUATION_THRESHOLD]: threshold,
-          [ALERT_EVALUATION_VALUE]: value,
-          [ALERT_REASON]: reason,
-          [ALERT_CONTEXT]: alertContext,
-          ...flattenAdditionalContext(rootLevelContext),
-        },
-      });
-
       if (actions && actions.length > 0) {
-        const indexedStartedAt = getAlertStartedDate(id) ?? startedAt.toISOString();
-        const relativeViewInAppUrl = getLogsAppAlertUrl(new Date(indexedStartedAt).getTime());
-
-        const viewInAppUrl = addSpaceIdToPath(
-          basePath.publicBaseUrl,
-          spaceId,
-          relativeViewInAppUrl
-        );
-
-        const sharedContext = {
-          timestamp: startedAt.toISOString(),
-          viewInAppUrl,
-        };
-
         asyncForEach(actions, async (actionSet) => {
```

Comment:
does this have to be async? I don't see any `await` inside the loop. Is there a missing `await`?

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/178680/commits/caa163f3d5455b6a09160abc491d9a870fcd208f